changelog/preview-flags.dd

```diff
@@ -0,0 +1,84 @@
+`-preview` and `-revert` flags have been introduced
+
+For a long time new features have been added with little oversight and thus
```

Comment:
"For a long time new features have been added on an ad-hoc basis and thus"

Reply:
"Command line switches adding or subtracting features of the D language have been normally added on an ad-hoc basis. Over time this has grown to:"